src/sentry/models/project.py

```diff
@@ -404,6 +406,12 @@ def transfer_to(self, organization):
 
         AlertRule.objects.fetch_for_project(self).update(organization=organization)
 
+        # Manually move over external issues to the new org
+        linked_groups = GroupLink.objects.filter(project_id=self.id).values_list("id", flat=True)
+        ExternalIssue.objects.filter(organization_id=old_org_id, id__in=linked_groups).update(
+            organization_id=organization.id
+        )
```

Comment:
The largest project has 20k rows here, not totally sure if that'd run fast enough before getting stomped... But most project have < 1k so maybe we don't really need to overoptimize this

Reply:
Would it be sufficient to add a limit of ~1k?

Reply:
Yeah, could potentially fetch up to 1k at a time until there are no left for that org

Reply:
Could you wrap the query in `RangeQuerySetWrapper` and then do a [bulk_update](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#bulk-update) on the chunk of `ExternalIssue`s? Kinda like what's done here:
https://github.com/getsentry/sentry/blob/13ddd83cc812ac758c5a493541e38700159a3cb2/src/sentry/tasks/auto_archive_issues.py#L71-L84